src/utils/pushnotification.utils.ts

```
@@ -0,0 +1,39 @@
+import { Lambda } from 'aws-sdk';
+import { SendNotificationToDevice } from '@src/types';
+
+const SEND_NOTIFICATION_LAMBDA_ENDPOINT = process.env.SEND_NOTIFICATION_LAMBDA_ENDPOINT;
```

Maybe we should throw an error here or log with `ALERT` so we get notified that this env is not set